paasta_tools/cli/cmds/push_to_registry.py

```diff
@@ -108,3 +136,43 @@ def paasta_push_to_registry(args):
             level='event',
         )
     return returncode
+
+
+def read_docker_registy_creds(registry_uri):
+    dockercfg_path = os.path.expanduser('~/.dockercfg')
```

Comment:
fwiw new versions of docker use `~/.docker/config.json` instead (though maybe still support this old path)